dbms/src/IO/Compression/ZlibInflatingReadBuffer.cpp

```diff
@@ -25,14 +25,15 @@ ZlibInflatingReadBuffer::ZlibInflatingReadBuffer(
     size_t alignment)
     : BufferWithOwnMemory<ReadBuffer>(buf_size, existing_memory, alignment)
     , in(in_)
+    , zstr()
```

Comment:
If we have `zstr()` here, maybe we can remove the lines from  `next_in` to `avail_out`?

Reply:
Actually those files are useless now, removed